### PR TITLE
[ticket/14117] Add core events to index.php to allow modifying birthdays list

### PR DESCRIPTION
**Pull request by rxu**
<a href="https://tracker.phpbb.com/browse/PHPBB3-14117">PHPBB3-14117</a>.

### CODE REVIEW

**Comment by rxu** [comment]
!set Event

File: phpBB/index.php [hunk 1]
```diff
@@ -132,33 +132,67 @@
...
+		$birthday_list[] = $birthday_username . (($birthday_age) ? " ({$birthday_age})" : '');
 	}
-	$db->sql_freeresult($result);
+
+	/**
+	* Event to modify the birthdays list
+	*
+	* @event core.index_modify_birthdays_list
+	* @var	array	birthdays		Array with the users birhtdays data
+	* @var	array	birthday_list	Array with the birthdays list items
```
Comment (Nicofuma) [inline phpBB/index.php]: It is only used for 3.0 compatibility. I don't think we should add it to the event (or at least repeat the deprecated comment here)

**Comment by rxu** [comment]
@Nicofuma Updated #3850.

**Comment by Nicofuma** [comment]
:+1:

**Pull request merged**